bin/update_configs.sh

```diff
@@ -0,0 +1,115 @@
+set -eu
+
+should_push=${1:-false}
+update_dev=${2:-true}
+update_prod=${2:-false}
+
+
+all_prod_sites=( \
```

Comment:
We have this list o a couple of place, but I can't think of a simple way to sync them.

Can you update this with the current list from [base-fork](https://github.com/greenpeace/planet4-base-fork/blob/develop/.circleci/config.yml#L207)? Plus the 3 EA sites missing from there.

Reply:
Seems like it makes sense to add it in this repo. Then we can adjust the NRO generator script to add the new site to the corresponding list, so that we're sure the generated site will be updated.